desci-server/Dockerfile

```
@@ -0,0 +1,31 @@
+FROM node:20.18.1-bullseye-slim
+
+VOLUME /root/.yarn
+
+RUN apt-get -qy update && apt-get -qy install openssl curl socat jq
+
+RUN mkdir /app && chown -R node:node /app && chown -R node:node /root && mkdir -p /root/.yarn/v6
+
+WORKDIR /app
+
+COPY --chown=node:node ./package.json ./yarn.lock .
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)